mobilecoind-json/tests/integration_test.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 The MobileCoin Foundation
+#
+# Integration test that uses mobilecoind-json to submit a transaction and check balance.
+
+import argparse
+import glob
+import json
+import logging
+import os
+import sys
+import time
+import urllib.request
+
+
+logging.basicConfig(stream = sys.stdout, level = logging.INFO, format="%(levelname)s:%(module)s:%(lineno)s: %(message)s")
+
+
+class MobilecoindJsonClient:
+    def __init__(self, base_url):
+        if not base_url.endswith("/"):
+            base_url += "/"
+        self.base_url = base_url
+
+    def request(self, url, data=None):
+        req = urllib.request.Request(self.base_url + url)
+
+        if data:
+            data = json.dumps(data)
+            data_bytes = data.encode("utf-8")
+
+            req.add_header("Content-Type", "application/json; charset=utf-8")
+            req.add_header("Content-Length", len(data_bytes))
+            response = urllib.request.urlopen(req, data_bytes)
+        else:
+            response = urllib.request.urlopen(req)
+
+        return json.loads(response.read())
+
+    def get_balance(self, monitor_id, subaddress_index=0):
+        response = self.request(f"monitors/{monitor_id}/subaddresses/{subaddress_index}/balance")
+        if "balance" not in response:
+            raise Exception(f"Invalid response: {response}")
+        return int(response["balance"])
+
+    def create_monitor(self, account_key, first_subaddress_index=0, num_subaddresses=10):
+        return self.request(f"monitors", {
+            "account_key": account_key,
+            "first_subaddress": first_subaddress_index,
+            "num_subaddresses": num_subaddresses,
+        })
+
+    def get_monitor(self, monitor_id):
+        return self.request(f"monitors/{monitor_id}")
+
+    def get_ledger_local(self):
+        return self.request(f"ledger/local")
+
+    def wait_for_monitor_to_sync(self, monitor_id, poll_interval=0.1):
```

Comment:
Nit: This is rather different from the other methods that forward to `self.request()`, so how about we move it below those methods?